.ci/conformance/download_tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+wget -O test_adaboost.py https://raw.githubusercontent.com/scikit-learn/scikit-learn/0.21.3/sklearn/ensemble/tests/test_weight_boosting.py
```

Comment:
suggest to make this a variable - "https://raw.githubusercontent.com/scikit-learn/scikit-learn"

make this as another variable - 0.21.3